openmetadata-ui/src/main/resources/ui/src/components/GithubStarCard/GithubStarCard.component.tsx

```diff
@@ -89,12 +87,12 @@ const GithubStarCard = () => {
 
   const githubPopup = useCallback(
     (show: boolean) => {
-      if (loggedInUserName && show) {
+      if (currentUser?.name && show) {
         fetchOpenMetaData();
       }
       setShowGithubStarPopup(show);
     },
-    [loggedInUserName, userCookieName]
+    [currentUser?.name, userCookieName]
   );
```

Comment:
why do we need this change? isn't it good to have this value in one place and use it across? I would recommend to keep old code!

Reply:
@ShaileshParmar11 We were occupying unnecessary memory space, by storing it in a variable, we can it directly.